Create_users_on_website.py

```diff
@@ -0,0 +1,36 @@
+"""
+Example Admas created on 6/29 office hours to create random email address and add a user.
+
+Example command:
+ $ python Create_users_on_website.py --number_of_users=5
+"""
+
+from woocommerce import API
+import uuid
+import argparse
+# import sys
+# arguments = sys.argv
+# number_of_users = arguments[1]
+parser = argparse.ArgumentParser(description='Process some integers.')
+parser.add_argument('--number_of_users', required=True,
+                    help='how many users to create')
+args = parser.parse_args()
+num_of_users = args.number_of_users
+
+wcapi = API(
+    url="http://localhost:8888/mysite2",
+    consumer_key="<add consumer key>",
+    consumer_secret="<add consumer secret>",
+    version="wc/v3",
+    timeout = 60
+)
+rand_pass = str(uuid.uuid4())
+for i in range(int(num_of_users)):
```

Comment:
Can you please log number of users created everytime a new one is creted.

Reply:
Added a counter to the script that outputs to the .log file and prints to the display.

Reply:
The counter works fine. You did great with that.
But could you just use 'i' for the counter? Isn't "i" just a number?

Reply:
replaced the counter with i. makes it  a lot cleaner and fewer lines of code. Is this the same i thats being used in the for loop?

Reply:
answering my own question: it is definitely the same i !